docs/guide/scheme.md

```diff
@@ -20,7 +20,84 @@ auth: {
   strategies: {
     local1: { _scheme: 'local', /* ... */ },
     local2: { _scheme: 'local', /* ... */ },
-    custom: { _scheme: '~/app/customStrategy.js', /* ... */ },
+    custom: { _scheme: '~/app/customScheme', /* ... */ },
   }
 }
 ```
+
+## Creating your own scheme
+Sometimes the included schemes doesn't match your requirements. Creating your own scheme will provide flexibility you need. You can create a new scheme from scratch or extend an existing scheme.
+
+> **Note** A list of available scheme methods will be added in future releases.
+
+Before creating custom schemes, you will need to add `@nuxtjs/auth` to `transpile`.
+
+`nuxt.config.js`
+```js
+build: {
+  transpile: ['@nuxtjs/auth']
```

Comment:
This should be already done by auth module to self-transpile:https://github.com/nuxt-community/auth-module/blob/dev/src/module/index.ts#L41

Reply:
The docs are for `master` branch, so it doesn't self-transpile :)

Reply:
I'm improving more docs for custom scheme, but after merge to `dev` branch, I'll remove the transpile part.

Reply:
Thanks :) BTW for master, providers are copied to `.nuxt` (which should be imported using relative path not NPM package)

Reply:
Hmm ok. I'll fix that.